mypy.ini

```diff
@@ -0,0 +1,41 @@
+# Global options:
```

Comment:
love this! 
Should add an entry in ``makefile.common`` to run it from any project?

Reply:
ok take it easy, it is not yet in a definitive state. it is still too much depending on VSCode. so currently I only have the start of the mypy configuration.

@ignapas : [mypy](https://mypy.readthedocs.io/en/stable/introduction.html) it is a static type checker